tests/test_seq_domain_red.py

```diff
@@ -171,13 +171,20 @@ def verify_bounds_in_range(new_bounds, global_bounds):
     trimmed_bounds = bounds_transformer._trim(new_bounds, global_bounds)
     assert (trimmed_bounds == np.array( [[-5, 5], [-10, 10]] )).all()
 
-    # test if both bounds for one parameter are beyond the global bounds
+    # test if both (upper/lower) bounds for a parameter exceed the global bounds
     new_bounds = np.array( [[-50, -20], [-10, 10]] )
     trimmed_bounds = bounds_transformer._trim(new_bounds, global_bounds)
     assert verify_bounds_in_range(trimmed_bounds, global_bounds)
 
-    # test if both bounds for one parameter are beyond the global bounds 
+    # test if both (upper/lower) bounds for a parameter exceed the global bounds 
     # while they are out of order
     new_bounds = np.array( [[-20, -50], [-10, 10]] )
     trimmed_bounds = bounds_transformer._trim(new_bounds, global_bounds)
     assert verify_bounds_in_range(trimmed_bounds, global_bounds)
+
+if __name__ == '__main__':
+    r"""
+    CommandLine:
+        python tests/test_seq_domain_red.py
+    """
+    pytest.main([__file__])
```

Comment:
this is consistent with the other test files.

Reply:
I think these should be removed, as they are not needed anymore, but it's good that you made sure it's consistent. Let's keep it like this for now, and we can delete them all in the future.

Reply:
I found them helpful when writing the tests, since I could just "run" the test files, but I understand if you decide to remove them.

Reply:
let's keep them for now.